dockers/requirements.txt

```diff
@@ -1,11 +1,14 @@
-guacamol @ git+https://github.com/PaccMann/guacamol.git@0.5.4
+guacamol==0.5.4
 matplotlib==3.0.2
-torch>=1.0.0
+torch>=1.0
 loguru==0.5.3
 joblib==0.12.5
-numpy==1.18
+numpy>=1.16.5
 tqdm==4.26.0
 cython==0.29
 nltk==3.4.5
 flake8==3.5.0
-molsets @ git+https://github.com/molecularsets/moses.git@7b8f83b21a9b7ded493349ec8ef292384ce2bb52
\ No newline at end of file
+tensorflow
+keras
+h5py
+moses @ git+https://github.com/PaccMann/moses.git@0.0.1
```

Comment:
Would it make sense to move Moses to gt4sd org first?

Reply:
I have just raised a new PR in the GT4SD org for Moses :)

Reply:
Should we point to Moses of GT4SD org in this requirements as most likely we'll archive the Moses repo here in Paccman?